Makefile

```diff
@@ -41,22 +41,22 @@ ifndef SR_XML
 endif
 
 build:
-	jbuilder build --dev \
+	dune build --dev \
```

Comment:
—dev is the default in dune. You need to add —release to the other dune build invocations